jupyter-docker/Dockerfile

```diff
@@ -43,13 +43,19 @@ ENV PYSPARK_DRIVER_PYTHON_OPTS notebook
 ENV PATH $SPARK_HOME:$SPARK_HOME/python:$SPARK_HOME/bin:$HAIL_HOME:$PATH
 ENV PYTHONPATH $PYTHONPATH:$HAIL_HOME/$HAILPYTHON:$HAIL_HOME/python:$SPARK_HOME/python:$SPARK_HOME/python/lib/py4j-0.10.3-src.zip:$JUPYTER_HOME/custom
 
+
 RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security jessie/updates main" > /etc/apt/sources.list \
  && apt-get update && apt-get -yq dist-upgrade \
  && apt-get install -yq --no-install-recommends \
+    nano \
+    #note ca-certificates has to be before wget and curl
+    ca-certificates \
     wget \
+    curl \
     maven \
     git \
```

Comment:
I wonder why we need git?

Reply:
we used to build hail from source. we don't any more -- there's a ticket in the backlog to optimize the docker image